test_results/README.md

```diff
@@ -1,3 +1,10 @@
 # Test Results
 
-A bunch of test results, these will likely change.
\ No newline at end of file
+CSV files contain summary statistics (mean, stdev, percentiles etc.) of various metrics across the course of each test:
+- `recv_bytes_per_second_*` - total bytes received by the container, per second.
+- `cpu_seconds_*` - total CPU seconds consumed by the container.  This is currently NOT expressed relative to the node (ex. percent CPU utilization).  CPU seconds on their own is a bit hard to interpret, need to make this better.
```

Comment:
This is still meaningful for the context of "how CPU efficient is the task", right? The fewer cpu seconds the better, right?